sdk/src/beta9/cli/machine.py

```diff
@@ -189,3 +191,27 @@ def delete_machine(service: ServiceClient, machine_id: str, pool: str):
         terminal.success(f"Deleted machine '{machine_id}' from pool '{pool}'")
     else:
         terminal.error(f"Error: {res.err_msg}")
+
+
+@management.command(
+    name="available-gpus",
+    help="Get available GPU types.",
+)
+@extraclick.pass_service_client
+def available_gpus(service: ServiceClient):
```

Comment:
@mernit Any thoughts on this interface to machine availability? Specifically on the name `available-gpus`

Reply:
I think it's fine @luke-lombardi @nickpetrovic

Reply:
Should the CLI command and the SDK function have the same name at least? They're different because I wanted to see if there was a preference of one vs the other.

Reply:
Yeah `machine.list()` should ideally list the available GPUs, maybe we should use that so its consistent